src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+import java.util.function.BiFunction
+
+enum class Operator(
+    private val symbol: String,
+    private val execute: BiFunction<Int, Int, Int>,
```

Comment:
크 `BiFunction` 을 이용하셔서 깔끔하고 명료하게 표현하셨네요! 👍 
조금 더 코틀린 감수성을 얹어보면 다른식으로 표현할 수 있는데요!
[SAM](https://kotlinlang.org/docs/fun-interfaces.html) 인데요!
이미 `BiFunction`이 SAM이지만 아래와 같이 표현하는 방법이 있습니다!
```suggestion
    private val execute: (Int, Int) -> Int,
```

이렇게 변경하시면 아래 PLUS, MINUS, MULTIPLY, DIVIDE에 있는 `BiFunction` 을 제거할 수 있답니다!

Reply:
저도 `코틀린 감수성` 매우 좋아합니다! 😃 
SAM은 자바 진영에서 `함수형 인터페이스`로 공부를 했었는데, 사실 SAM(Single Abstract Method) 의미 자체가 동일한 말이군요? 😃